include/seqan/graph_msa/graph_align_tcoffee_library.h

```diff
@@ -887,6 +1021,62 @@ appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
 
 //////////////////////////////////////////////////////////////////////////////
 
+template<typename TString, typename TSpec, typename TSize2, typename TSpec2, typename TScore, typename TSegmentMatches, typename TScoreValues, typename TDistance, typename TAlignConfig, typename TSize3>
+inline void
+appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
+                     String<TSize2, TSpec2> const& pList,
+                     TScore const& score_type,
+                     TSegmentMatches& matches,
+                     TScoreValues& scores,
+                     TDistance& dist,
+                     TAlignConfig const& ac,
+                     TSize3 const & bandWidth,
+                     GlobalPairwiseLibrary,
+                     Banded) 
```

Comment:
Please update the argument order. First modified arguments than the unchanged ones.